clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/description/ScaleCloudFoundryServerGroupDescription.java

```diff
@@ -24,12 +25,11 @@
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class ScaleCloudFoundryServerGroupDescription extends AbstractCloudFoundryServerGroupDescription {
-  @Nullable
-  private Integer instanceCount;
+  ServerGroup.Capacity capacity;
```

Comment:
You're right, that should probably be `@Nullable`. Given the way the front-end works I don't currently see a scenario where it will be null but from a `defence against the dark arts` POV it's better to write it that way.

Reply:
For the record, I'm thinking more about automation we build separately from the UI.